js/translations.js

```diff
@@ -3,6 +3,23 @@ function updateContent() {
   $('body').localize();
 }
 const defaultLanguage = 'en';
+const LANGUAGES = {
+  'ca': 'Català',
+  'de': 'Deutsch',
+  'en': 'English',
+  'es-ES': 'Español',
+  'fr': 'Français',
+  'hu': 'magyar',
+  'it': 'Italiano',
+  'ja': '日本語',
+  'ko': '한국어',
+  'nl': 'Nederlands',
+  'pt-BR': 'Português',
+  'ru': 'Русский',
+  'zh-CN': '简体中文',
+  'zh-TW': '繁體中文',
+  'th-TH': 'ไทย',
```

Comment:
```suggestion
  'es': 'Español',
  'fr': 'Français',
  'hu': 'magyar',
  'it': 'Italiano',
  'ja': '日本語',
  'ko': '한국어',
  'nl': 'Nederlands',
  'pt-BR': 'Português',
  'ru': 'Русский',
  'th': 'ไทย',
  'zh-CN': '简体中文',
  'zh-TW': '繁體中文'
```
If the browser locale is set to `es`, `locales/es.json` will be loaded, but the selector will select `en`. Make sure that language codes and file names are consistent.